testcases/unit_testcases/test_blockchain_address_validation.py

```diff
@@ -0,0 +1,28 @@
+import unittest
+
+from utils.blockchain import is_valid_ethereum_address, is_valid_cardano_address
+
+
+class TestAddressValidation(unittest.TestCase):
+    def test_ethereum_address_validation(self):
+        valid_ethereum_address = "0x4e934Fc82mmn84aBe1C1b2556b9bF3055dBdd43c"
```

Comment:
_**This address can't be correct.**_

There cannot be any other letters and numbers in the address below

All Ethereum addresses **consist only of**:
Numbers: `0 1 2 3 4 5 6 7 8 9`
Letters: `a b c d e f A B C D E F`

Reply:
Good point, removed this test case entirely since we will use the web3`s built-in function now